### DISCUSSION TITLE
dryrun on Fastlane

### DISCUSSION THREAD

**Discussion by IvicaUdovicic** [fastlane/fastlane#22232] [Q&A]
I'm testing some lanes on a project, and I need a way to test the lane without actually uploading it to google app store, is there a way to use dryrun so i don't upoload it, will need this option for iOS too later. 

Thanks in advance

**Comment by carolgrey98** [comment]
Yes, you can perform a "dry run" of your deployment or release process for both Android and iOS without actually uploading your app to the Google Play Store or the Apple App Store.
[NextCare Urgent Care](https://www.nextcare-urgentcare.com)

**Reply by IvicaUdovicic** [reply]
Could you explain how? when I try to use --dryrun label it does not recognize it.

**Reply by Jaredbeyers23** [reply]
@IvicaUdovicic any luck with this? Similar issue for me
